grpc/src/main/java/com/linecorp/armeria/server/grpc/UnframedGrpcErrorHandlerBuilder.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2024 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server.grpc;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.Set;
+
+import org.curioswitch.common.protobuf.json.MessageMarshaller;
+
+import com.google.common.collect.ImmutableSet;
+
+import com.linecorp.armeria.common.annotation.Nullable;
+
+public final class UnframedGrpcErrorHandlerBuilder {
+
+    private static final Set<UnframedGrpcErrorResponseType> DEFAULT_RESPONSE_TYPES =
+            ImmutableSet.copyOf(UnframedGrpcErrorResponseType.values());
+
+    @Nullable
+    private UnframedGrpcStatusMappingFunction statusMappingFunction;
+
+    @Nullable
+    private MessageMarshaller jsonMarshaller;
+
+    private Set<UnframedGrpcErrorResponseType> responseTypes = DEFAULT_RESPONSE_TYPES;
+
+    public UnframedGrpcErrorHandlerBuilder jsonMarshaller(MessageMarshaller jsonMarshaller) {
+        this.jsonMarshaller = requireNonNull(jsonMarshaller, "jsonMarshaller");
+        return this;
+    }
+
+    public UnframedGrpcErrorHandlerBuilder statusMappingFunction(
```

Comment:
Because we accept the mandatory property (status mapping function is a mandatory property) when a user calls `of*()` or `builder()`, so we don't need to add this method, i.e.
```java
UnframedGrpcErrorHandler
  .builder(mappingFunc)
  ...
  .build();
// or
UnframedGrpcErrorHandler.of(mappingFunc);
```